ols/constants.py

```diff
@@ -51,9 +51,43 @@ class QueryValidationMethod(StrEnum):
 GPT35_TURBO_1106 = "gpt-3.5-turbo-1106"
 GPT35_TURBO = "gpt-3.5-turbo"
 
-# Model configs
+
+# Token related constants
 DEFAULT_CONTEXT_WINDOW_SIZE = 8000
 DEFAULT_RESPONSE_TOKEN_LIMIT = 500
+MINIMUM_CONTEXT_TOKEN_LIMIT = 1
+
+DEFAULT_TOKENIZER_MODEL = "cl100k_base"
+
+
+# RAG related constants
+
+# This is used to decide how many matching chunks we want to retrieve as context.
+# (in descending order of similarity between query & chunk)
+# Currently we want to fetch best matching chunk, hence the value is set to 1.
+# If we want to fetch multiple chunks, then this value will increase accordingly.
+
+# This also depends on chunk_size used during index creation,
+# if chunk_size is small, we need to set a higher value, so that we will get
+# more context. If chunk_size is more, then we need to set a low value as we may
+# end up using too much context. Precise context will get us better response.
+RAG_CONTENT_LIMIT = 1
+
+# Once the chunk is retrived we need to check similarity score, so that we won't
+# pick any random matching chunk.
+# Currently we use L2/KNN based FAISS index. And this cutoff signifies distance
+# between chunk and query in vector space. Lower distance means query & chunk are
+# more similar. So lower cutoff value is better.
+
+# If we set a very high cutoff, then we may end up picking irrelevant chunks.
+# And if we set a very low value, then there is risk of discarding all the chunks,
+# as there won't be perfect matching chunk. This also depends on embedding model
+# used during index creation/retrieval.
+
+# This will be reversed, if we use cosine/dot product or any other metric for
+# which higher score means two vectors are more similar. Additionaly we also need
+# to change if-condition (> cutoff or < cutoff)
+RAG_SIMILARITY_CUTOFF = 0.5
```

Comment:
given that the nature of the cutoff value(high is good vs low is good) changes depending on the type of scoring we're doing, i think it would make sense to include the scoring algorithm in the constant name.  That way it is less likely someone will reuse the constant with a different scoring algorithm, resulting in unexpected behavior.

also please indicate the valid range for this value.